internal/pkg/utils/ioutil/writer.go

```diff
@@ -61,7 +61,9 @@ func (w *Writer) Flush() (err error) {
 	return nil
 }
 
-func (*Writer) Close() error { return nil }
+func (w *Writer) Close() error {
+	return w.Flush()
```

Comment:
Tu chybalo volanie `Flush`, sice to nerobilo ziadne problemy, ale radsej som to fixol.